src/components/permissions/routes.ts

```diff
@@ -211,7 +211,7 @@ const routeGetPrefs = async (request: any, reply: any) => {
 
 // Unique name/email/organisation/other check
 const routecheckUnique = async (request: any, reply: any) => {
-  const { type, value, table, field } = request.query
+  const { type, value, table, field, caseInSensitive } = request.query
```

Comment:
That's great, just one super-nitpicky thing: I think it should be `caseInsensitive` since "insensitive" is a single unbroken work in standard English.

Reply:
Of course, that's great. Great attention to detail.